### ISSUE TITLE
determine the details of `VersionResponse`

### ISSUE THREAD

**Issue by haiyanmeng** [mrunalp/ocid#2]
@mrunalp , before starting writing the server code, I want to double check I understand the info source of each field.

```
type VersionResponse struct {
    // The version of the gRPC API
    Version *string `protobuf:"bytes,1,opt,name=version" json:"version,omitempty"`
    // The name of the container runtime.
    RuntimeName *string `protobuf:"bytes,2,opt,name=runtime_name" json:"runtime_name,omitempty"`
    // The version of the container runtime.
    RuntimeVersion *string `protobuf:"bytes,3,opt,name=runtime_version" json:"runtime_version,omitempty"`
    // The API version of the container runtime.
    RuntimeApiVersion *string `protobuf:"bytes,4,opt,name=runtime_api_version" json:"runtime_api_version,omitempty"`
    XXX_unrecognized  []byte  `json:"-"`
}
```

`Version`: this should be the `Rev` field inside Godeps/Godeps.json, right?

`RuntimeName` should be: `runc`

`RuntimeVersion` should be from the result of `runc -v`.

```
[hmeng@localhost runc]$ runc -v
runc version 1.0.0-rc1
commit: 4eb8c2fb1dcb10fa3bf9bd7031f3a25a8ce2fef6
spec: 1.0.0-rc1
```

`RuntimeApiVersion`: I am not sure this is applicable for runc.

`XXX_unrecognized`: We can leave it empty.

**Comment by mrunalp** [comment]
Let API version be v1alpha1. The rest sounds good. We want to eventually accept runtime as argument to server so it is pluggable.

**Comment by haiyanmeng** [comment]
Sounds good.

**Comment by haiyanmeng** [comment]
Fix by #4 .
